Review request

donnybrilliant opened the pull request:
Only basic layout done, but would still like a review so I know where to go from here.

Fermain reviewed (commented):
What is done so far is of a very high standard, but the key things missing still:

- Profile page (with requirements)
- Deeper SASS customisation

Your layout is great, now make it pop!

Fermain commented:
Also, I don't see all your files in the PR. Looks like you already committed some to main which is quite normal. Abi posted a video on what to do about this yesterday. I will try to find it now.

donnybrilliant commented:
Yes. Profile and customization is in the works.

After i merged it into main, i reverted it and made a new pull request.
I updated my preferences so that I need a review from someone else to merge into main. Guess this is standard?

Will look for the video.
